lib/tutorials/en_US/getting-started.md

```diff
@@ -83,51 +89,50 @@ The `method` parameter can be any valid HTTP method, array of HTTP methods, or a
 
 We've proven that we can start a simple Hapi app with our Hello World application. Next, we'll use a plugin called **inert** to serve a static page. Before you begin, stop the server with **CTRL + C**.
```

Comment:
I know this isn't part of your edit but we should make all instances of Hapi lowercased to hapi.